docs/components/connectors/out-of-the-box-connectors/kafka-inbound.md

```diff
@@ -0,0 +1,143 @@
+---
+id: kafka-inbound
+title: Kafka Consumer Connector
+sidebar_label: Kafka Consumer Connector
+description: "The Kafka Consumer Connector allows you to connect your BPMN service with Kafka. Learn how to create a Kafka Consumer Connector and make it executable."
+---
+
+The **Kafka Consumer Connector** allows you to connect your BPMN service with [Kafka](https://kafka.apache.org/).
+
+## Prerequisites
+
+To use the **Kafka Consumer Connector**, you need to have a Kafka instance with configured bootstrap server.
+It is highly recommended not to expose your sensitive data as plain text, but rather use Camunda secrets. Follow our documentation on [managing secrets](/components/console/manage-clusters/manage-secrets.md) to learn more.
+
+## Create a Kafka Consumer Connector task
+
+1. Start building your BPMN diagram with a **Start Event** building block.
+2. Change its template to a Kafka Consumer.
```

Comment:
I may explain how to do this -- do I click somewhere specific in the UI?

Reply:
@sbuettner what do you think? From my point of view, that is nothing specific for Kafka

Reply:
@christinaausley Should we maybe have a dedicated page on how to user inbound connectors? It seems tedious to describe the steps for each and every connector.

Reply:
Lets improve this in the future.